build.gradle

```diff
@@ -54,3 +56,7 @@ snyk {
     arguments = '--all-sub-projects'
     severity = 'high'
 }
+
+licenseReport {
+    allowedLicensesFile = new File("$projectDir/.circleci/allowed-licenses.json")
```

Comment:
as it is more specific to Gradle I would perhaps move it to buildSrc/

Reply:
done